docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -2997,6 +3025,34 @@ Must contain the certificates's Common Name, and SHA-256 fingerprint in the form
     You must enable client authentication using the
     [`---rpc-http-tls-client-auth-enabled`](#rpc-http-tls-client-auth-enabled) option.
 
+### `rpc-http-tls-protocol`, `rpc-http-tls-protocols`
+
+=== "Syntax"
+
+    ```bash
+    --rpc-http-tls-protocol=<protocolName>[, <protocolName>...]
+    ```
+
+=== "Example"
+
+    ```bash
+    --rpc-http-tls-protocol=TLSv1.3,TLSv1.2
+    ```
+
+=== "Environment variable"
+
+    ```bash
+    BESU_RPC_HTTP_TLS_PROTOCOL=TLSv1.3,TLSv1.2
+    ```
+
+=== "Configuration file"
+
+    ```bash
+    rpc-http-tls-protocol=["TLSv1.3,TLSv1.2"]
```

Comment:
I think you want to have a list of strings. Did you try it?
    rpc-http-tls-protocol=["TLSv1.3","TLSv1.2"]